lib/tools/ms_erref.js

```diff
@@ -88,10 +88,10 @@ var facilities = {
     "code":"FACILITY_SECURITY"
   , "message":"The source of the error code is the Security API layer."
   }
-, 9:{
-    "code":"FACILITY_SSPI"
-  , "message":"The source of the error code is the Security API layer."
-  }
+// , 9:{
+//     "code":"FACILITY_SSPI"
+//   , "message":"The source of the error code is the Security API layer."
+//   }
```

Comment:
I don't want to remove it, you got dupplicate keys that break sctrict mode execution. Now that I read it again, I'm not sure which one of the dupplicates is "hidden" in non-strict mode and so never existed in fact. I'll recheck.